app/controllers/catalog_controller.rb

```diff
@@ -9,6 +10,18 @@ class CatalogController < ApplicationController
   end
 
   configure_blacklight do |config|
+    # default advanced config values
+    config.advanced_search ||= Blacklight::OpenStructWithHashAccess.new
+    # config.advanced_search[:qt] ||= 'advanced'
+    config.advanced_search[:url_key] ||= 'advanced'
+    config.advanced_search[:query_parser] ||= 'edismax'
+    config.advanced_search[:form_solr_parameters] ||= {
+      # 'facet.field' => ['language_facet_ssim'], # will add 'library', 'location', 'material_type'
+      'facet.limit' => -1, # return all facet values
```

Comment:
Subjects listed on advanced search seems to be listing only 20 subjects because of `limit: 20` here: https://github.com/psu-libraries/psulib_blacklight/blob/%2346-Advanced-Search/app/controllers/catalog_controller.rb#L112. 

`'facet.limit' => -1` supposed to be taking take of this, I guess? @cdmo could you also check pls? I tried adding `"f.subject_topic_facet_ssim.facet.limit" => -1,` to `config.advanced_search[:form_solr_parameters]` and it returned all. 

Similar behavior for author and language dropdowns as well since there is also a limit defined for these facets.

Reply:
For now I propose that we just comment out these custom configs. Especially the byte-order one.

We will likely have to do some more customization, like displaying the number of items for a given facet.